docs/release/release_0_4_15.md

```diff
@@ -0,0 +1,131 @@
+# napari 0.4.15
+
+We're happy to announce the release of napari 0.4.15!
+napari is a fast, interactive, multi-dimensional image viewer for Python.
+It's designed for browsing, annotating, and analyzing large multi-dimensional
+images. It's built on top of Qt (for the GUI), vispy (for performant GPU-based
+rendering), and the scientific Python stack (numpy, scipy).
+
+
+For more information, examples, and documentation, please visit our website:
+https://github.com/napari/napari
+
+## Highlights
+
+
+## New Features
+
+- Add edge_width per-point and edge_width_is_relative (#3999)
+- Support error reporting via napari-error-reporter (#4055)
+
+## Improvements
+
+- avoid repeated computation of layers.extent when adding a new Labels layer (#4037)
+- Add style encoding for strings (#4051)
+- Cache layerlist expensive properties (#4054)
+- Prevent errors in qt/test_plugin_widgets from installed plugins (#4058)
+- Add a `recurse` option to `EventedModel.update` (#4062)
+- Only add progress widget on worker start (#4068)
+- minor refactor of Layer._slice_indices (#4072)
+- Improve missing widget name error for npe2 plugins (#4080)
+- Update shapes color mapping when changing property (#4081)
+- reduce NumPy overhead in ScaleTranslate and Affine transform calls (#4094)
+- Remove remaining references to requirements.txt (#4105)
+- Try to standardise API as list of strings. (#4107)
+
+## Documentation
+
+- Use cells3d image in README example (#3823)
+- update event tables in the in-depth guides (#4030)
+- add quick start (#4041)
+- New documentation infrastructure (#4047)
+- Add Grzegorz to 0.4.14 release notes (#4061)
+- remove `--checkout npe2` instructions for cookiecutter in documentation (#4092)
+- Clarify that napari should be installed (#4101)
+- Move workshop out of tutorials into usage (#4123)
+- Refactor layers content (#4124)
+- Organize tutorials sections (#4137)
+- Add release management docs (#4139)
+- Use napari-sphinx-theme in docs build (#4147)
+- Add new core developers (#4157)
```

Comment:
Potential highlight? 🙂

Reply:
Well, both were added before 0.4.15, so it's a bit weird... LOL We just forgot to update the docs

Reply:
okay by me either way here :)